NuGet.Config

```diff
@@ -5,6 +5,7 @@
     <clear />
     <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
+    <add key="roslyn" value="https://dotnet.myget.org/f/roslyn/api/v3/index.json" />
```

Comment:
CLI-Deps wasn't caught up yet. Will remove once it's back up to speed, in a future PR.